docs/UserGuide.md

```diff
@@ -143,13 +143,13 @@ Shows a message explaining how to access the help page.
 
 Format: `help`
 
-### Adding a student: `add`
+### Adding a student: `add -s`
 Adds a new student to TutorAid.
 
 Format: `add -s sn/STUDENT_NAME [sp/STUDENT_PHONE] [pn/PARENT_NAME] [pp/PARENT_PHONE]`
 
 Examples:
-* `add -s sn/John Does sp/81234567 pn/Mrs Doe pp/91234567`
+* `add -s sn/John Does sp/81234567 pn/Mrs Doe pp/91234567` adds a student with name John Does, student phone 81234567, parent's name Mrs Doe and parent's phone 91234567.
```

Comment:
Maybe can write it this way to make it more complete: `adds a student with name 'John Does', student phone number '81234567', parent's name 'Mrs Doe' and parent's phone number '91234567'`